### PR TITLE
Roadmap 2021

### PR DESCRIPTION
**Pull request by felixfaassen**
Implements #103 

- Add order on roadmap timeline. Latest developments on top, oldest below (NL/EN)
- Add roadmap news article (NL/EN)
- Updated roadmap highlight on main page

### CODE REVIEW

File: src/pages/news/2021-06-24-roadmap.en.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+path: "/en/news/2021-06-24-roadmap/"
+date: "2021-06-24"
+title: "Roadmap Signalen July 2021 - December 2021"
+lang: en
+template: default
+---
+
+## Roadmap Signalen July 2021 - December 2021
+The product steering group has been busy working with all community members of Signalen to
+determine which functional improvements they would like to see within Signalen.
```
Comment (AlbaRoza) [inline src/pages/news/2021-06-24-roadmap.en.md:11]: Is there an unnecessary line break in between "to" and "determine"?

File: src/pages/news/2021-06-24-roadmap.en.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+---
+path: "/en/news/2021-06-24-roadmap/"
+date: "2021-06-24"
+title: "Roadmap Signalen July 2021 - December 2021"
+lang: en
+template: default
+---
+
+## Roadmap Signalen July 2021 - December 2021
```
Comment (AlbaRoza) [inline src/pages/news/2021-06-24-roadmap.en.md:9]: Title with two ## ?

File: src/pages/news/2021-06-24-roadmap.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+path: "/news/2021-06-24-roadmap/"
+date: "2021-06-24"
+title: "Roadmap Signalen Juli 2021 - December 2021"
+lang: nl
+template: default
+---
+
+## Roadmap Signalen Juli 2021 - December 2021
+De afgelopen tijd is de product steering groep druk bezig geweest om met alle community leden van Signalen
+vast te stellen welke verbeteringen en doorontwikkelingen zij graag zouden zien ten aanzien van Signalen.
```
Comment (AlbaRoza) [inline src/pages/news/2021-06-24-roadmap.md:11]: Line break in between "Signalen" and "vast"?

File: src/pages/news/2021-06-24-roadmap.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
+---
+path: "/news/2021-06-24-roadmap/"
+date: "2021-06-24"
+title: "Roadmap Signalen Juli 2021 - December 2021"
+lang: nl
+template: default
+---
+
+## Roadmap Signalen Juli 2021 - December 2021
```
Comment (AlbaRoza) [inline src/pages/news/2021-06-24-roadmap.md:9]: Is it # or ## for the title?

File: src/pages/news/2021-06-24-roadmap.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+De afgelopen tijd is de product steering groep druk bezig geweest om met alle community leden van Signalen
+vast te stellen welke verbeteringen en doorontwikkelingen zij graag zouden zien ten aanzien van Signalen.
+
+Het thema is dan ook om ons in de 2e helft van dit jaar te gaan richten op het gebruikersvriedenlijker maken
+van Signalen voor de mensen die de meldingen afhandelen. Hieronder volgt op hoofdlijnen aan welke onderdelen we zullen gaan werken:
```
Comment (AlbaRoza) [inline src/pages/news/2021-06-24-roadmap.md:14]: "maken"

line break

"van Signalen" ?

File: src/pages/news/2021-06-24-roadmap.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+| 2a     | Filtering |  Refactor |  large    |
+| 2b     | Filtering |  Autocomplete toevoegen adres |  small    |
+| 3a     | Gebruikersvriendelijker beheer | Routeringsregels in Django admin |  small    |
+| 3b     | Gebruikersvriendelijker beheer | Machine learning, onderzoek nodig |  large   |
+| 3c     | Gebruikersvriendelijker beheer | Aanvullende vragen implementer in front-end|  large   |
+| 4     | Verrijken data dump | Verrijken data dump|  small/medium  |
+| 4     | Notificeren van behandelaars| Mailtje versturen / Instelbaar maken |  medium  |
+
+---
+<br/>
```
Comment (AlbaRoza) [inline src/pages/news/2021-06-24-roadmap.md:30]: Does the news section recognize HTML?

File: src/pages/news/2021-06-24-roadmap.en.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+| 2a | Filtering | Refactor | large |
+| 2b | Filtering | Autocomplete add address | small |
+| 3a | More user-friendly management | Routing rules in Django admin | small |
+| 3b | More user-friendly management | Machine learning, research needed | large |
+| 3c | More user-friendly management | Additional questions implementer in front-end| large |
+| 4 | Enrich data dump | Enrich data dump| small/medium |
+| 4 | Notify practitioners| Send mail / Make configurable | medium |
+
+---
+<br/>
```
Comment (AlbaRoza) [inline src/pages/news/2021-06-24-roadmap.en.md:29]: HTML in news section?

**Review by AlbaRoza — COMMENTED** [review]
Made some changes, let's see what you think, @felixfaassen

File: src/pages/news/2021-06-24-roadmap.en.md (hunk 2 above)
Comment (felixfaassen) [reply src/pages/news/2021-06-24-roadmap.en.md:9]: yes to stay inline with site formatting

File: src/pages/news/2021-06-24-roadmap.md (hunk 4 above)
Comment (felixfaassen) [reply src/pages/news/2021-06-24-roadmap.md:9]: ## to remain consistent with styling

File: src/pages/news/2021-06-24-roadmap.en.md (hunk 7 above)
Comment (felixfaassen) [reply src/pages/news/2021-06-24-roadmap.en.md:28]: yes so the table is not added to the text. This is allowed in this setup

File: src/pages/index.en.js [hunk 8]
```diff
@@ -111,9 +111,9 @@ const IndexPage = ({ location }) => (
       <Timeline>
         <Timeline.Container align="left">
           <Timeline.Content>
-            <Span fontSize="0.9rem">Q4 2020</Span>
-            <Heading as="h3" fontSize="1.5rem">Ready for production</Heading>
-            <p>Delivery of a municipal neutral version of the Signalen software. Signalen is ready for production for the municipal participants from the Kopgroep.</p>
+            <Span fontSize="0.9rem">May 2021</Span>
+            <Heading as="h3" fontSize="1.5rem">Signalen is used in production at the municipality of 's-Hertogenbosch</Heading>
+            <p>Signalen was taken into production at the beginning of May in the municipality of 's-Hertogenbosch.</p>
```
Comment (Ainali) [inline src/pages/index.en.js:116]:
```suggestion
            <p>Signalen was taken into production at the beginning of March in the municipality of 's-Hertogenbosch.</p>
```

File: src/pages/index.en.js [hunk 9]
```diff
@@ -111,9 +111,9 @@ const IndexPage = ({ location }) => (
       <Timeline>
         <Timeline.Container align="left">
           <Timeline.Content>
-            <Span fontSize="0.9rem">Q4 2020</Span>
-            <Heading as="h3" fontSize="1.5rem">Ready for production</Heading>
-            <p>Delivery of a municipal neutral version of the Signalen software. Signalen is ready for production for the municipal participants from the Kopgroep.</p>
+            <Span fontSize="0.9rem">May 2021</Span>
```
Comment (Ainali) [inline src/pages/index.en.js:114]:
```suggestion
            <Span fontSize="0.9rem">March 2021</Span>
```

File: src/pages/index.js [hunk 10]
```diff
@@ -113,9 +113,9 @@ const IndexPage = ({ location }) => (
       <Timeline>
         <Timeline.Container align="left">
           <Timeline.Content>
-            <Span fontSize="0.9rem">Q4 2020</Span>
-            <Heading as="h3" fontSize="1.5rem">Klaar voor productie</Heading>
-            <p>Oplevering gemeente-neutrale versie Signalen-software. Signalen is klaar voor productie bij de gemeentelijke deelnemers uit de Kopgroep.</p>
+            <Span fontSize="0.9rem">Mei 2021</Span>
```
Comment (Ainali) [inline src/pages/index.js:116]:
```suggestion
            <Span fontSize="0.9rem">Maart 2021</Span>
```

File: src/pages/index.js [hunk 11]
```diff
@@ -113,9 +113,9 @@ const IndexPage = ({ location }) => (
...
+            <Heading as="h3" fontSize="1.5rem">Signalen is in gebruik bij 's-Hertogenbosch</Heading>
+            <p>Signalen is begin mei in product genomen in de gemeente 's-Hertogenbosch.</p>
```
Comment (Ainali) [inline src/pages/index.js:118]:
```suggestion
            <p>Signalen is begin maart in product genomen in de gemeente 's-Hertogenbosch.</p>
```

**Comment by felixfaassen** [comment]
@Ainali @AlbaRoza Thank you for reviewing

File: src/pages/roadmap.en.js [hunk 12]
```diff
@@ -42,77 +42,104 @@ const RoadmapPage = ({ location }) => (
...
+        </Timeline.Content>
+      </Timeline.Container>
+
+        <Timeline.Container align="right">
           <Timeline.Content>
-            <Span fontSize="0.9rem">2018</Span>
-            <Heading as="h3" fontSize="1.5rem">Start of development of SIA</Heading>
-            <p>The Municipality of Amsterdam starts the development of Signalen in Amsterdam (SIA) with the Department of Services and the Datalab. SIA is open source and therefore freely usable for others.</p>
-            <ResponsiveImage src={startOntwikkeling} />
+            <Span fontSize="0.9rem">May 2021</Span>
```
Comment (Ainali) [inline src/pages/roadmap.en.js:56]:
```suggestion
            <Span fontSize="0.9rem">March 2021</Span>
```

File: src/pages/roadmap.js [hunk 13]
```diff
@@ -42,77 +42,104 @@ const RoadmapPage = ({ location }) => (
...
+        </Timeline.Content>
+      </Timeline.Container>
+
+        <Timeline.Container align="right">
           <Timeline.Content>
-            <Span fontSize="0.9rem">2018</Span>
-            <Heading as="h3" fontSize="1.5rem">Start ontwikkeling SIA</Heading>
-            <p>Gemeente Amsterdam start de ontwikkeling van Signalen in Amsterdam (SIA) met de afdeling Dienstverlening en het Datalab. SIA is opensource en daarmee vrij bruikbaar voor anderen.</p>
-            <ResponsiveImage src={startOntwikkeling} />
+            <Span fontSize="0.9rem">Mei 2021</Span>
```
Comment (Ainali) [inline src/pages/roadmap.js:56]:
```suggestion
            <Span fontSize="0.9rem">Maart 2021</Span>
```

File: src/pages/roadmap.js [hunk 14]
```diff
@@ -42,77 +42,104 @@ const RoadmapPage = ({ location }) => (
...
+            <Heading as="h3" fontSize="1.5rem">Signalen in 's-Hertogenbosch</Heading>
+            <p>Signalen is begin mei in product genomen in de gemeente 's-Hertogenbosch.</p>
```
Comment (Ainali) [inline src/pages/roadmap.js:58]:
```suggestion
            <p>Signalen is begin maart in product genomen in de gemeente 's-Hertogenbosch.</p>
```

File: src/pages/news/2021-06-24-roadmap.md [hunk 15]
```diff
@@ -0,0 +1,32 @@
...
```
Comment (Ainali) [inline src/pages/news/2021-06-24-roadmap.md:29]: Is this inline html really needed? It looks quite good without it to me.

File: src/pages/news/2021-06-24-roadmap.en.md [hunk 16]
```diff
@@ -0,0 +1,31 @@
...
```
Comment (Ainali) [inline src/pages/news/2021-06-24-roadmap.en.md:28]: Is this inline html really needed? It looks quite good without it to me.

File: src/pages/news/2021-06-24-roadmap.md (hunk 15 above)
Comment (felixfaassen) [reply src/pages/news/2021-06-24-roadmap.md:29]: yes as the line comes to close to the table which I think makes it not readable

File: src/pages/news/2021-06-24-roadmap.en.md (hunk 16 above)
Comment (felixfaassen) [reply src/pages/news/2021-06-24-roadmap.en.md:28]: yes as the line comes to close to the table which I think makes it not readable

File: src/pages/news/2021-06-24-roadmap.en.md [hunk 17]
```diff
@@ -0,0 +1,31 @@
...
+More detailed information can be found on our [project board on Github](https://github.com/orgs/Signalen/projects/) with user stories labeled as “Roadmap 2021”.
```
Comment (Ainali) [inline src/pages/news/2021-06-24-roadmap.en.md:29]: This link doesn't render like a proper link for me, not when I build it locally nor when just viewing the file here in GitHub. It is the same problem on the Dutch version.

File: src/pages/roadmap.en.js [hunk 18]
```diff
@@ -42,77 +42,104 @@ const RoadmapPage = ({ location }) => (
     </Container>
     <Container>
       <Timeline>
-      <Timeline.Container align="right">
+
+      <Timeline.Container align="left">
+        <Timeline.Content>
+          <Span fontSize="0.9rem">Juli 2021</Span>
+          <Heading as="h3" fontSize="1.5rem">Roadmap 2nd half 2021</Heading>
+          <p>The roadmap of Signalen has been defined by the product steering group. The community is now looking how to finance these developments. Read more about the <a href="/en/news/2021-06-24-roadmap/">roadmap 2021</a></p>
```
Comment (Ainali) [inline src/pages/roadmap.en.js:50]:
```suggestion
          <p>The roadmap of Signalen has been defined by the product steering group. The community is now looking how to finance these developments. Read more about the <a href="/en/news/2021-06-24-roadmap/">roadmap 2021</a>.</p>
```

File: src/pages/roadmap.js [hunk 19]
```diff
@@ -42,77 +42,104 @@ const RoadmapPage = ({ location }) => (
     </Container>
     <Container>
       <Timeline>
-      <Timeline.Container align="right">
+
+      <Timeline.Container align="left">
+        <Timeline.Content>
+          <Span fontSize="0.9rem">Juli 2021</Span>
+          <Heading as="h3" fontSize="1.5rem">Roadmap 2e helft 2021 van Signalen</Heading>
+          <p>De roadmap van Signalen is vastgesteld door de product steering groep. Er wordt nu binnen community gekeken naar budget om deze ontwikkelingen te financieren. Lees hier meer over <a href="/news/2021-06-24-roadmap/">de roadmap 2021</a></p>
```
Comment (Ainali) [inline src/pages/roadmap.js:50]:
```suggestion
          <p>De roadmap van Signalen is vastgesteld door de product steering groep. Er wordt nu binnen community gekeken naar budget om deze ontwikkelingen te financieren. Lees hier meer over <a href="/news/2021-06-24-roadmap/">de roadmap 2021</a>.</p>
```

File: src/pages/news/2021-06-24-roadmap.en.md (hunk 17 above)
Comment (felixfaassen) [reply src/pages/news/2021-06-24-roadmap.en.md:29]: fixed

**Review by Ainali — APPROVED** [review]

**Pull request merged**